src/tests/unit/doughs-dao.test.js

```diff
@@ -179,4 +184,52 @@ describe('test doughs dao', () => {
       });
     });
   });
+  context('getDoughById', () => {
+    let result;
+    beforeEach(() => {
+      connectionStub = sinon.stub().returns(executeReturn);
+      doughsDao = proxyquireDoughsDao(
+        connectionStub,
+        serializeDoughsStub,
+        serializeDoughStub,
+      );
+    });
+    context('when called with an integer id', () => {
+      beforeEach(async () => {
+        result = doughsDao.getDoughById(1);
```

Comment:
Again, ID must be a string. My bad I didn't catch it on the last PR, but you should modify your [DAO method](https://github.com/osu-mist/pizza-api/blob/6de21a4f650721e595469d986cb0a168cf3ae442/src/api/v1/db/oracledb/doughs-dao.js#L248-L264) to take string instead of a number. Test data should be modified as well.

Reply:
Oops yeah I'll get this fixed, didn't updated the test logic and the docs after changing that in the openapi spec.